### PR TITLE
Typo correctness for some UI files

### PR DESCRIPTION
**Pull request by shrinktofit**

### CODE REVIEW

**Comment by shrinktofit** [comment]
Request review from @JoneLau

File: cocos/ui/assembler/label/ttf.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export const ttf: IAssembler = {
 
         vbuf.set(vData, vertexOffset);
 
-        // fill indice data
+        // fill index data
```
Comment (JoneLau) [inline cocos/ui/assembler/label/ttf.ts:117]: indices 吧

File: cocos/ui/assembler/sprite/simple.ts [hunk 2]
```diff
@@ -137,7 +137,7 @@ export const simple: IAssembler = {
 
         vbuf.set(vData, vertexOffset);
 
-        // fill indice data
+        // fill index data
```
Comment (JoneLau) [inline cocos/ui/assembler/sprite/simple.ts:140]: indices

File: cocos/ui/assembler/utils.ts [hunk 3]
```diff
@@ -87,7 +87,7 @@ export function fillMeshVertices3D (node: INode, renderer: UI, renderData: Rende
         vertexOffset += 4;
     }
 
-    // fill indice data
+    // fill index data
```
Comment (JoneLau) [inline cocos/ui/assembler/utils.ts:90]: indices

**Comment by shrinktofit** [comment]
Close this since delayed to 1.1. @JoneLau

**Review by JoneLau — CHANGES_REQUESTED** [review]

**Pull request closed without merging**